backend/src/services/video/pythonHelpers/tests/test_channel_registration.py

```
@@ -0,0 +1,176 @@
+    # Two unrelated random frames → no real common structure; any peak found
+    # must be rejected (implausible / low confidence) rather than applied.
+    a = np.random.RandomState(3).rand(128, 128) * 1000
+    b = np.random.RandomState(99).rand(128, 128) * 1000
+    dy, dx, _ = estimate_translation(a, b)
+    assert (dy, dx) == (0, 0)
+
+
+def test_low_signal_frame_falls_back_to_zero():
+    ref = _synthetic_frame(4)
+    flat = np.full_like(ref, 200.0)  # dark/flat channel: no edges to match
+    dy, dx, conf = estimate_translation(ref, flat)
```

Ruff (0.15.20)

[warning] Unpacked variable `conf` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)